partners/core.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Core",
+  "description": "Avalanche Core Wallet is a secure, user-friendly wallet for AVAX and other Avalanche tokens. It offers easy asset management, staking, and dApp integration, making it suitable for all users.",
```

Comment:
I think it should stop at "Avalanche Core Wallet is a secure, user-friendly wallet for AVAX and other Avalanche tokens."